tests/test_mockgun.py

```diff
@@ -269,6 +269,19 @@ def test_find_by_sub_entity_field(self):
         )
         self.assertEqual(len(items), 1)
 
+    def test_find_with_none(self):
+        """
+        Ensures comparison with multi-entity fields and None works.
+        """
+        items = self._mockgun.find("PipelineConfiguration", [["users", "is", None]], ["users"])
+        self.assertEqual(len(items), 1)
+        self.assertEqual(items[0]["users"], [])
+
+        items = self._mockgun.find("PipelineConfiguration", [["users", "is_not", None]], ["users"])
```

Comment:
This would deserve a second test. One for the case of `is` and a second for `is_not`.

It will increase the readability of the tests